src/index.js

```diff
@@ -0,0 +1,266 @@
+"use strict";
+
+// Constants
+const state = {
+  realTempValue: 70,
+  realTempValueColor: 'orange',
+  gardenTempValue: 70,
+  gardenTempValueColor: 'orange',
+  landscape: '🌸🌿🌼__🌷🌻🌿_☘️🌱_🌻🌷',
+  // sky: 'sunny',
+  name: 'Seattle',
+  weather: 'clear',
+  weatherIconCode: '01d',
+  weatherDescription: 'clear sky',
+};
+
+const tempProperties = [ 
+{   temp: 80, 
+    color: 'red',
+    landscape: '🌵__🐍_🦂_🌵🌵__🐍_🏜_🦂'
+  },
+{   temp: 70,
+    color: 'orange',
+    landscape: '🌸🌿🌼__🌷🌻🌿_☘️🌱_🌻🌷'
+  },
+{   temp: 60,
+    color: 'yellow',
+    landscape: '🌾🌾_🍃_🪨__🛤_🌾🌾🌾_🍃'
+  },
+{   temp: 50,
+    color: 'green',
+    landscape: '🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲' 
+  },
+{   temp: 49,
+    color: 'teal',
+    landscape: '🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲'
+  }
+];
+
+const skyOptions = {
+  'sunny': '☀️☀️☀️☀️☀️☀️',
+  'cloudy': '☁️☁️☁️🌤☁️☁️☁️',
+  'rainy': '🌧🌈⛈🌧💧⛈🌧🌦💧🌧',
+  'snowy': '🌨❄️🌨❄️🌨❄️🌨❄️🌨🌨',
+};
+
+// Helper Functions
+const updateState = (data) =>{
+  for (let key of Object.keys(data)){
+    state[key] = data[key];
+  }
+};
+
+const HandleCityNameError = () =>{
+  const cityErrorMessageModal = document.getElementById('cityErrorDialog');
+  cityErrorMessageModal.showModal();
+  let errorData = {
+    name: '??',
+    weather: '??',
+    weatherIconCode: '??',
+    weatherDescription: '??',
+    realTempValue: '??'
+  };
+  updateState(errorData);
+  document.getElementById('realTempValue').textContent = errorData.realTempValue;
+  changeRealTempValueColor();
+  changeRealWeatherDetails();
+}
+
+const closeErrorModal = () =>{
+  const cityErrorMessageModal = document.getElementById('cityErrorDialog');
+  cityErrorMessageModal.close();
+};
+
+//Wave 2
+const changeGardenTempValueColorAndLandscape = () =>{
+  let data = {};
+  for (const idx in tempProperties){
+    let temp = tempProperties[idx].temp;
+    if (temp === 49 && state.gardenTempValue<= temp){
```

Comment:
It's important to leave spaces between comparison operators so it's easy to see the parts of the expression at a quick glance.
```suggestion
    if (temp === 49 && state.gardenTempValue <= temp){
```